server/index.ts

```diff
@@ -17,6 +17,25 @@ import { schema, TypeOf } from '@osd/config-schema';
 import { PluginInitializerContext, PluginConfigDescriptor } from '../../../src/core/server';
 import { SecurityPlugin } from './plugin';
 
+const validateAuthType = (value: string[]) => {
+  const supportedAuthTypes = [
+    '',
+    'basicauth',
+    'jwt',
+    'openid',
+    'saml',
+    'proxy',
+    'kerberos',
+    'proxycache',
+  ];
+
+  value.forEach((authVal) => {
+    if (!supportedAuthTypes.includes(authVal.toLowerCase())) {
+      return `allowed auth.type are ${supportedAuthTypes}`;
```

Comment:
Pointing out supported auth types are great! But I would also add a clear message "Some of the configured auth types are not supported" or something like this if pointing out the specific ones that are not supported is not the best way here.

Reply:
Addressed.
![image](https://user-images.githubusercontent.com/92183424/196005226-b5c74fd8-0dd8-401e-baa6-a318ff86ebef.png)